Dockerfile

```
@@ -23,52 +18,47 @@ RUN apt-get update && apt-get install -y  \
     quantum-espresso      \
   && rm -rf /var/lib/apt/lists/*
 
-# needed for jupyterlab
+# Install what is needed for Jupyter Lab.
 RUN apt-get update && apt-get install -y \
      nodejs                \
      npm                   \
   && rm -rf /var/lib/apt/lists/*
 
-
-# Quantum-Espresso Pseudo Potentials
+# Quantum-Espresso Pseudo Potentials.
 WORKDIR /opt/pseudos
 RUN base_url=http://archive.materialscloud.org/file/2018.0001/v3;  \
 wget ${base_url}/SSSP_efficiency_pseudos.aiida;                    \
 wget ${base_url}/SSSP_precision_pseudos.aiida;                     \
 chown -R root:root /opt/pseudos/;                                  \
 chmod -R +r /opt/pseudos/
 
-# install packages that are not in the aiidalab meta package
-# 'fastentrypoints' is to fix problems with aiida-quantumespresso plugin installation
-RUN pip3 install --upgrade         \
-    'fastentrypoints'              \
-    'tornado==5.1.1'               \
+
+# Install Python packages needed for AiiDA lab.
+RUN pip install --upgrade         \
```

This might be why tornado is updated?
```suggestion
RUN pip install          \
```

---

Indeed, removing upgrade fixes a lot of stuff, also the AiiDA version problem that we discussed offline

---

I've put it all in one RUN command